2001-3000/LIT2892RepCh173.xml

```diff
@@ -25,7 +25,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </sourceDesc>
          <sourceDesc>
             <listWit>
-               <witness/>
+               <witness corresp="BNFabb88">B.A. 88</witness>
```

Comment:
just as with names and places - either do not insert any text in the element
`<witness corresp="BNFabb88"/>` - then the standard reference will be printed
or make sure that the text you insert corresponds to what you want to see, so the correct name/shelfmark etc.

Since you are not adding any information to the shelfmark in this case it is better to not insert any text at all.

If you wanted to add something beside the shelfmark (e.g. the ms contains many texts and you want to point to the precise locus etc) then of course you can add text, but it should correspond to our standard
e.g. `<witness corresp="BNFabb88">Paris, BnF, Éthiopien d'Abbadie 88, ff. **-**</witness>`

Reply:
Thank you! Here I wanted to quote what Chaine gives as a witness in his repertoire in the way he wrote it, that is why it is B.A. 88. I think the user can profit from this information. What would be the right way to inform the reader that  B.A. 88. corresponds to BNFabb88?

Reply:
The list of witnesses is not meant to quote catalogues, it is a list of mss used for the edition provided in the record [and must be standard](https://betamasaheft.eu/Guidelines/?id=witness). 

Unless you want to add a folio or a similar information do not provide any text.
If you for some reason need to quote the catalogue verbatim (though I do not see any sense in informing the reader that Chaîne called the ms B.A. 88 under the work record - this is an information to add to the manuscript description, if you think it is important) it must be done elsewhere in the record

Reply:
I am not quoting the catalogue, I am quoting his repertory of texts, B.A. 88  works as a siglum to my understanding. But I will delete this information as you suggest it.